tests/python/gaia-ui-tests/gaiatest/tests/functional/ftu/test_ftu_skip_tour.py

```diff
@@ -40,19 +40,27 @@ def test_ftu_skip_tour(self):
         ssid = self.testvars['wifi']['ssid']
         psk = self.testvars['wifi'].get('psk')
         keymanagement = self.testvars['wifi'].get('keyManagement')
+        share_data_default = True
```

Comment:
Sorry to not have replied to you. It's a single point of change when they inevitably flip-flop on this decision, rather than having to change lines 115 and 126 separately.

Also, it's usually better to put a value in an explicit variable when it's meaningful, rather than using a magic True/False where otherwise not obvious--even when only using it once. The value is the variable name, for self-commenting.